README.md

```diff
@@ -170,22 +170,3 @@ $ oc create -f your-operator-source.yaml
 Once created, the Marketplace operator will use the `OperatorSource` to download your operator artifact from the app registry and display your operator offering in the Marketplace UI.
 
 You can also access private AppRegistry repositories via an authenticated `OperatorSource`, which you can learn more about [here](docs/how-to-authenticate-private-repositories.md).
-
-## Running End to End (e2e) Tests
```

Comment:
Please add a link to the e2e doc here instead of removing the full section.

Reply:
Addressed in the updated commit.